### PR TITLE
Update manifest for Spilo.

### PR DESCRIPTION
**Pull request by alexeyklyukin**
Includes S3 WAL shipping and base backups.

### CODE REVIEW

File: kubernetes/spilo.yaml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+                            }
+                          ]
+                         },
+                         "topologyKey": "kubernetes.io/hostname" 
+                    }
+                  ]
+                 }
+            }
+    spec:
+      serviceAccountName: operator
```
Comment (alexeyklyukin) [inline kubernetes/spilo.yaml]: Hm, probably not. I don't want to remove it, but I will add a comemnt, explaining, what does it do and why do we need it.

**Comment by alexeyklyukin** [comment]
👍

**Comment by CyberDem0n** [comment]
:+1:

**Comment by alexeyklyukin** [comment]
👍

**Pull request merged**